api/gyresources/endpoints/DiseaseController.py

```diff
@@ -110,17 +110,6 @@ def get(self):
                 response=sqlerr,
                 message="SQL error: "+str(sqlerr),
                 status=500)
-        except Exception as err:
```

Comment:
If you use only SQLAlchemy exception treatment, you'll have [this error](https://travis-ci.org/CapivaraProjects/GYResources/jobs/344045790#L759).<br />

Please, use generic Exception or be carefully using specific exceptions which can be used